api/repo.go

```diff
@@ -77,6 +77,7 @@ func CreateRepo(c *gin.Context) {
 	// capture middleware values
 	u := user.Retrieve(c)
 	allowlist := c.Value("allowlist").([]string)
+	defaultLimit := c.Value("defaultLimit").(int64)
```

Comment:
Should this be updated to `defaultBuildLimit` instead? I know `defaultTimeout` is present elsewhere, but adding the additional context to additions feels like a good change.

Reply:
Yeah I was trying to be consistent with `defaultTimeout` because I have an idea for refactor effort

That being said, I agree with you so I'll make this change now 😃

Reply:
This should be addressed 👍